SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Reporting security issues
+
+To report security issues, please reach out to the [maintainers](https://github.com/lima-vm/lima/blob/master/MAINTAINERS.md) directly via e-mail.
```

Comment:
Do we want to move the maintainers file too?

Reply:
`MAINTAINERS.md` is not one of the [supported file types](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) for "community health files". So you can create it in the `.github` repo, but it will not be shared across to other repos.